src/Bicep.LangServer/Snippets/Templates/res-aks-cluster.bicep

```diff
@@ -1,32 +1,32 @@
 ﻿// Kubernetes Service Cluster
-resource aksCluster 'Microsoft.ContainerService/managedClusters@2021-03-01' = {
-  name: ${1:'aksCluster'}
+resource ${1:aksCluster} 'Microsoft.ContainerService/managedClusters@2021-03-01' = {
+  name: ${2:'name'}
   location: resourceGroup().location
   identity: {
     type: 'SystemAssigned'
   }
   properties: {
-    kubernetesVersion: '${2|1.19.7,1.19.6,1.18.14,1.18.10,1.17.16,1.17.13|}'
-    dnsPrefix: ${3:'dnsprefix'}
+    kubernetesVersion: '${3|1.19.7,1.19.6,1.18.14,1.18.10,1.17.16,1.17.13|}'
+    dnsPrefix: ${4:'dnsprefix'}
     enableRBAC: true
     agentPoolProfiles: [
       {
         name: 'agentpool'
-        count: ${4:3}
-        vmSize: ${5:'Standard_DS2_v2'}
+        count: ${5:3}
+        vmSize: ${6:'Standard_DS2_v2'}
         osType: 'Linux'
         mode: 'System'
       }
     ]
     linuxProfile: {
-      adminUsername: ${6:'adminUserName'}
+      adminUsername: ${7:'adminUserName'}
       ssh: {
         publicKeys: [
           {
-            keyData: ${7:'keyData'}
+            keyData: ${8:'UPDATEME'}

```

Comment:
I am guessing we are going to update- UPDATEME later?

Reply:
No, this is to make it clear to the consumer that this value must be replaced/updated.

Reply:
Got it . Thanks!

Reply:
> I have placeholders values that correspond to their property names (e.g. `dnsPrefix: 'dnsPrefix'`), unless it's a property that MUST be changed or parameterized in order to deploy. In that case, I use 'UPDATEME'

Reply:
Open to suggestions on this one, if it doesn't make sense.

Reply:
Should we just treat it like other fields? We have fake username, password and IP address in other places, so the snippet is not deployable as is anyways. So is there any value in flagging only some fields?

Reply:
The other property values should match the property name. Those _should_ be deployable (but not recommended). If someone wants to quickly deploy a snippet for dev/test purposes they would be able to only fill in what's required.

Reply:
@alex-frankel any thoughts on this? Would REQUIRED be a better option?

Reply:
+1 on `REQUIRED` but ok with `UPDATEME`

Reply:
I liked `REQUIRED` as well. Just pushed updates for this.